packages/react-components/react-tabster/src/tabster-types-6.0.1-do-not-use.ts

```diff
@@ -0,0 +1,1214 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * WARNING! ATTENTION! ACHTUNG! WARNING! ATTENTION! ACHTUNG! WARNING! ATTENTION! ACHTUNG!
+ * WARNING! ATTENTION! ACHTUNG! WARNING! ATTENTION! ACHTUNG! WARNING! ATTENTION! ACHTUNG!
+ *
+ * Do not use anything from this file. It is a snapshot of the older Tabster typings exposed by a mistake.
+ * The exposed typings should have been removed, but we don't do it in minor versions to avoid breaking changes.
+ * Everything reexported from this file as react-tabster/TabsterTypes is marked as deprecated and shouldn't
+ * be used anywhre.
+ *
+ * WARNING! ATTENTION! ACHTUNG! WARNING! ATTENTION! ACHTUNG! WARNING! ATTENTION! ACHTUNG!
+ * WARNING! ATTENTION! ACHTUNG! WARNING! ATTENTION! ACHTUNG! WARNING! ATTENTION! ACHTUNG!
+ */
+
+export const TabsterAttributeName = 'data-tabster';
```

Comment:
all of this boilerplate is really necessary in order to avoid braking changes ? 

that's quite a lot of code which also adds significant size to every v9 package, which is something we pointed out multiple times but got no response on this topic. cc @ling1726 

https://github.com/microsoft/fluentui/pull/30823#discussion_r1535438856
https://github.com/microsoft/fluentui/pull/30823#discussion_r1535449998

This PR 

<img width="821" alt="image" src="https://github.com/microsoft/fluentui/assets/1223799/8279d40e-32ad-4624-ae6b-54e9d195df1b">

Reply:
This particular file is a snapshot of accidentally exposed typings and it is not actually used anywhere (so, it shouldn't affect the bundle).

Not every Tabster bump increases the bundle size. I remember some were even decreasing it. This one comes with new features (and new features are driven by the component/partners needs). And it is not fair to sum the numbers in the above table. Because even if you use all of the above components together, there is only one Tabster in the bundle.

Regardless, I will spend some time going through Tabster code before the next bump. We cannot reduce it to zero, but something likely can be optimized in the bundle size regard.

Reply:
And above you link to the comments of yet unmerged PR. So it's not the increase from that PR plus increase of this PR, it's just this one. Yes, we do grow. Mostly because of new requirements (or when a bugfix takes more logic to be properly fixed). But again, I will look into optimizing/cleaning up.

Reply:
I've commented in the other PR about the bundle size as well. To not be blamed for ignoring the concerns (I was on vacation at the time of the other PR).

Reply:
I have double-checked. The bundle size mostly comes from a fix (https://github.com/microsoft/tabster/commit/c9626769359ad7b518446f962d6eb483c8ef6d4c) and a feature (https://github.com/microsoft/tabster/commit/ad23b8ca20d8b0c720aecd3bc439d630597962ad). I don't see anything accidental in the bundle change.